hoot-core/src/main/cpp/hoot/core/scoring/GraphComparator.cpp

```diff
@@ -72,13 +72,13 @@ GraphComparator::GraphComparator(OsmMapPtr map1, OsmMapPtr map2) :
       _mean(0.0),
       _ci(-1.0),
       _s(-1.0),
-      _maxGraphCost(0.),
```

Comment:
`_maxGraphCost` is no longer a member variable because it wasn't actually being used outside of the class or across iterations so it could be a local variable per iteration and is then passed to a few different functions.